tests/tests.js

```diff
@@ -862,6 +882,96 @@ describe('Convert Lat/Lon to UTM', function(){
       chai.assert.equal(-2583444, parseInt(coords[1]));
       chai.assert.equal(38, coords[2]);
     });
+    describe('around Arizona in the United States', function(){
+        it('should return easting=500000; northing=3762155; zone=12', function(){
+          var coords = [];
+          converter.LLtoUTMwithNS(34, -111, coords);
+          chai.assert.equal(500000, parseInt(coords[0]));
+          chai.assert.equal(3762155, parseInt(coords[1]));
+          chai.assert.equal(12, coords[2]);
+          chai.assert.equal('N', coords[3]);
+        });
+      });
+      describe('around Prescott/Chino Valley in Arizona', function(){
+        it('should return easting=362289; northing=3818618; zone=12', function(){
+          var coords = [];
+          converter.LLtoUTMwithNS(34.5, -112.5, coords);
+          chai.assert.equal(362289, parseInt(coords[0]));
+          chai.assert.equal(3818618, parseInt(coords[1]));
+          chai.assert.equal(12, coords[2]);
+          chai.assert.equal('N', coords[3]);
+        });
+      });
+      describe('immediately around Prescott city in Arizona', function(){
+        it('should return easting=365575; northing=3823561; zone=12', function(){
+          var coords = [];
+          converter.LLtoUTMwithNS(34.545, -112.465, coords);
+          chai.assert.equal(365575, parseInt(coords[0]));
```

Comment:
❓ Roughly half a degree of lat equates to a difference of 134425 in the easting? That seems large. And .035 in long to a diff of 4943?

Reply:
Yep it is in Meters and half a degree of lat is pretty huge.

Reply:
Makes more sense to me now. Prescott being 134km north of PHX is totally reasonable. 👍